azure-ipam/main.go

```diff
@@ -22,9 +22,9 @@ func main() {
 func executePlugin() error {
 	// logger config
 	loggerCfg := &logger.Config{
-		Level:            "debug",
-		OutputPaths:      "var/log/azure-ipam.log",
-		ErrorOutputPaths: "var/log/azure-ipam.log",
+		Level:                "debug",
+		Filename:             "var/log/azure-ipam.log",
```

Comment:
should be an absolute path?

Reply:
If you don't start it with "/" then it will consider it relative to "/" 
It was previously written like this and I didn't change it.

Reply:
lets fix this and have absolute path

Reply:
> If you don't start it with "/" then it will consider it relative to "/"

It will consider it relative to the current working directory, which _happens_ to be `/`. That is not guaranteed.